src/streaming/utils/TTMLParser.js

```diff
@@ -62,19 +62,15 @@ function TTMLParser() {
 
     /**
      * Drill down into JS representation of TTML Doc to look for tts:FontFamily styling properties.
-     * Match the given fontFamily names against active DVB font downloads and prefix the fontFamily
-     * names so they are unique to dashjs and don't clash with other browser/local fontFamily names.
+     * Cover for some missing functionality in imscJS v1.1.4 to correctly replace default to monospaceSerif.
+     * Default is a reserved word in CSS so this can result in no fontFamily being correctly applied.
+     * This is covered further in the imscJS Github repository, PR #251 and issue #250 
+     * TODO: This can be removed when imscJS v1.1.5 is released and used
      * @param {object} imsc1doc - JS Representation of TTML Doc
-     * @param {array} dvbFonts - Active DVB Font Downloads
-     * @returns {object} - JS Representation of TTML Doc with prefixed font families
+     * @returns {object} - JS Representation of TTML Doc with fixed fontFamily
      * @private
      */
-    function _addDvbFontFamilyPrefix(imsc1doc, dvbFonts) {
-
-        // Get original, unprefixed names
-        const fontFamilies = dvbFonts.map(fontInfo => {
-            return fontInfo.fontFamily.replace(Constants.DASHJS_DVB_FONT_PREFIX,'');
-        });
+    function _addFontFamilyParsingFix(imsc1doc) {
 
         // Go through nested contents and looks for tts:fontFamily styling to prefix
         function recursivelyPrefixFontFamilies (el) {
```

Comment:
maybe rename to "recursivelyTrimFontFamilies" or similar since it is no longer prefixing?